pkg/config/config.go

```diff
@@ -220,6 +237,12 @@ func (c *Config) saveToPath(configPath string) error {
 	return nil
 }
 
+// SaveToPath serializes the config struct and writes it to a specific path.
+// If configPath is empty, it uses the default path.
+func (c *Config) SaveToPath(configPath string) error {
```

Comment:
Consider renaming `saveToPath` to `SaveToPath` instead of wrapping it in a new function.